packages/core/src/app/checkout/AnalyticsEvents.d.ts

```diff
@@ -0,0 +1,19 @@
+export declare enum GuestCheckoutEvents {
+    CheckoutLoadSuccess = "Checkout load success",
+    DetailEntryBegan = "Detail entry began",
+    AccountButtonClick = "Account lookup button click",
+    BoltButtonExists = "Bolt recognized button rendered",
+    BoltButtonClicked = "Bolt recognized button clicked",
+    ShippingEntered = "Shipping details fully entered",
+    ShippingComplete = "Shipping method step complete",
+    BillingEntered = "Billing details entered",
+    PaymentEntered = "Payment details entered",
```

Comment:
```suggestion
    ShippingDetailsEntered = "Shipping details fully entered",
    ShippingComplete = "Shipping method step complete",
    BillingDetailsEntered = "Billing details entered",
    PaymentDetailsEntered = "Payment details entered",
```

Reply:
oh I just kept these short for ease of reference in the code

Reply:
up to you, not going nitpick